docs/conversion/sakai_20_oracle_conversion.sql

```diff
@@ -143,3 +143,7 @@ CREATE TABLE GOOGLEDRIVE_USER (
 ALTER TABLE SAKAI_PERSON_T ADD PHONETIC_PRONUNCIATION VARCHAR2(255) DEFAULT NULL;
 -- END SAK-41812
 
+-- SAK-42371
+ALTER TABLE rbc_evaluation ADD status INTEGER NOT NULL;
```

Comment:
`INTEGER` is not a valid Oracle datatype. You'll want a `NUMBER` of some sort. See https://docs.oracle.com/cd/B28359_01/server.111/b28318/datatype.htm#CNCPT1834